ensure_tiller_installed.go

```diff
@@ -351,6 +351,8 @@ func (c *Client) EnsureTillerInstalledWithValues(ctx context.Context, values []s
 				// Fall through as we need to install Tiller.
 				installTiller = true
 				return nil
+			} else if IsTooManyResults(err) {
+				return backoff.Permanent(err)
```

Comment:
Can we mask the error given to `backoff.Permanent`? Then we do not miss out on the location in the stack.

Reply:
Thx good catch! Done.